cmd/root.go

```diff
@@ -48,6 +50,8 @@ func init() {
 	rootCmd.AddCommand(NewAnalyzeCmd(logger))
 	rootCmd.AddCommand(NewTestCommand(logger))
 	rootCmd.AddCommand(NewVersionCommand())
+	rootCmd.AddCommand(discover.NewDiscoverCommand(logger))
+	rootCmd.AddCommand(generate.NewGenerateCommand(logger))
```

Comment:
What if we have `discover.go` and `generate.go` in `cmd` folder (not inside subfolders) and then inside them we refer to  `asset_generation/discover` and `asset_generation/generate`?  Just to use the same style of other commands.

The results will be something like 

```suggestion
	rootCmd.AddCommand(NewDiscoverCommand(logger))
	rootCmd.AddCommand(NewGenerateCommand(logger))
```

Reply:
I use the structure [suggested](https://github.com/spf13/cobra/blob/main/site/content/user_guide.md#organizing-subcommands) by Cobra itself. And it makes sense because variable names can overlap unless they have a very unique prefix/postfix to discern them.